Core/FileLoaders/LocalFileLoader.cpp

```diff
@@ -15,23 +15,25 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
-#include <cstdio>
 #include "file/file_util.h"
 #include "Common/FileUtil.h"
 #include "Core/FileLoaders/LocalFileLoader.h"
+#ifdef _WIN32
+#include "Common/CommonWindows.h"
+#endif
 
 LocalFileLoader::LocalFileLoader(const std::string &filename)
 	: fd_(0), f_(nullptr), filesize_(0), filename_(filename) {
+    // FIXME: perhaps at this point we should just use plain open?
 	f_ = File::OpenCFile(filename, "rb");
+	fd_ = fileno(f_);
```

Comment:
This segfaults immediately for me on Windows.  This occurs when `f_` is `nullptr`, which happens when the file loader gets pointed at a directory (which is used for "virtual iso" games and also may be used for homebrew and demos, etc.)

Moving the `fileno()` call down below the null check fixes this and everything else seems fine on Windows.  Might be an issue on non-Windows too.

-[Unknown]